src/components/UserManagement/UserTableData.jsx

```diff
@@ -9,24 +9,32 @@ import { faCopy } from '@fortawesome/free-solid-svg-icons';
 import { toast } from 'react-toastify';
 import { boxStyle } from 'styles';
 import { connect } from 'react-redux';
+// import { useHistory } from "react-router-dom";
 import { formatDate } from 'utils/formatDate';
 
 /**
  * The body row of the user table
  */
 const UserTableData = React.memo(props => {
   const [isChanging, onReset] = useState(false);
-  const canAddDeleteEditOwners = props.hasPermission('addDeleteEditOwners');
+  // const history = useHistory();
+  const canAddDeleteEditOwners = hasPermission('addDeleteEditOwners');
+  // const canAddDeleteEditOwners = props.hasPermission('addDeleteEditOwners');
 
+  console.log("user table", props)
```

Comment:
```suggestion
  const canAddDeleteEditOwners = props.hasPermission('addDeleteEditOwners');
```
Remove commented out code and console.logs.
The original usage of `hasPermission()` was correct. The way you changed it makes it so it returns a thunk (which will always equate to true) instead of the desired bool.